Feature/blui 3525 spacer examples (#577)
brightlayer-ui/react-component-library: docs/src/componentDocs/Spacer/markdown/SpacerExamples.mdx

import { SpacerWithFlex } from '../examples/SpacerWithFlex';
import { SpacerWithPixel } from '../examples/SpacerWithPixel';

# Spacer with Flex

> The `<spacer>` adds space between elements. It accepts a flex prop that determines how much relative space to consume.

Reviewer: ```suggestion
The `<Spacer>` adds space between elements. It accepts a `flex` prop that determines how much relative space to consume.
```

Author: verbiage updated to match NG doc description

> For a more contextual example, see how a `<spacer>` can be used to separate multiple [`<DrawerNavGroup>`](/components/drawer/drawer-nav-group/examples).

Reviewer: ```suggestion
For a more contextual example, see how a `<Spacer>` can be used to separate multiple [`<DrawerNavGroup>`](/components/drawer/drawer-nav-group/examples).
```

<SpacerWithFlex />

> # Spacer with Pixels.

Reviewer: ```suggestion
# Spacer with Pixels
```

> The `<spacer>` also accepts a height and width prop that specifies the number of pixels to consume for static layouts.

Reviewer: ```suggestion
The `<Spacer>` also accepts `height` and `width` props that specify the number of pixels to consume for static layouts.
```
